index.bs

```diff
@@ -2743,6 +2743,27 @@ and makes it clear when the state described by the flags is reset.
 
 <!-- TODO: add examples -->
 
+<h3 id="monkey-patching">Avoid monkey patching</h3>
+A <dfn export>monkey patch</dfn> layers new functionality on top of an existing specification in a way that extends, overrides, or otherwise modifies the underlying specification's behavior.
+Monkey patching <strong>is considered bad practice and should be avoided</strong> for the reasons listed below.
+
+An example of monkey patching would be: specification A defines an algorithm to do something, then specification B overrides or modifies that algorithm with some new functionality.
+
+As such, monkey patching (wrongly!) presupposes that underlying functionality cannot, and will not, be changed. This can lead to several problems:
+
+  * If the underlying specification changes, the monkey patched text may no longer apply thus causing it to break or lead to confusion.
+  * If several specifications monkey patch part of another spec, the order in which the patches are applied can lead to inconsistent behavior.
+  * The original specification authors may not be aware of the monkey patched text in your spec. This misses an opportunity to receive guidance on reaching consensus on a fix. Or worse, the behavior that is being patched is interrelated to so other issue or behavior, making the overall situation worse.
+  * An engineers doing code maintenance may read underlying specification and inadvertently revert the monkey patched behavior assuming it is a bug.
+
+<h4 id="cant-avoid-monkey-patching">If you need to monkey patch</h4>
+Sometimes monkey patching is unavoidable (e.g., early in the design phase of a new specification). If you find yourself needing to monkey patch a specification, make sure you:
+
+ 1. File a bug, or create a fix, for the underlying specification. This will help you understand what the problem is and how to fix it, and maybe even alleviate you from even creating the monkey patch in the first place.
```

Comment:
```suggestion
 1. File a bug, or create a fix, for the existing specification. This will help you understand what the problem is and how to fix it, and maybe even alleviate you from even creating the monkey patch in the first place.
```